src/zepto.js

```diff
@@ -690,18 +690,19 @@ var Zepto = (function() {
     },
     css: function(property, value){
       if (arguments.length < 2) {
-        var computedStyle, element = this[0]
-        if(!element) return
-        computedStyle = getComputedStyle(element, '')
-        if (typeof property == 'string')
-          return element.style[camelize(property)] || computedStyle.getPropertyValue(property)
-        else if (isArray(property)) {
+        var element = this[0]
+        if (typeof property == 'string') {
+          if (!element) return
+          return element.style[camelize(property)] || getComputedStyle(element, '').getPropertyValue(property)
+        } else if (isArray(property)) {
+          if (!element) return
           var props = {}
           $.each(property, function(_, prop){
-            props[prop] = (element.style[camelize(prop)] || computedStyle.getPropertyValue(prop))
+            props[prop] = (element.style[camelize(prop)] || getComputedStyle(element, '').getPropertyValue(prop))
```

Comment:
This calls `getComputedStyle(element, '')` in a loop for every property name, but it will always have the same result, so calling it many times is wasteful. Can you cache it outside of the loop in a `computedStyle` variable like it was before? Thanks